apps/ucp-ui/cypress.config.ts

```diff
@@ -3,6 +3,7 @@ const { defineConfig } = require("cypress");
 module.exports = defineConfig({
   e2e: {
     baseUrl: "http://localhost:3000/",
+    retries: 1,
```

Comment:
Is this because of the rate limit issue?

Reply:
Kind of, but having a retry is generally a healthy practice.